stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -10,19 +10,71 @@
 package com.facebook.stetho.inspector.elements.android;
 
 import android.view.View;
+import android.view.ViewDebug;
+import com.facebook.stetho.common.Accumulator;
+import com.facebook.stetho.common.LogUtil;
 import com.facebook.stetho.common.StringUtil;
 import com.facebook.stetho.common.android.ResourcesUtil;
-import com.facebook.stetho.inspector.elements.AttributeAccumulator;
 import com.facebook.stetho.inspector.elements.AbstractChainedDescriptor;
+import com.facebook.stetho.inspector.elements.AttributeAccumulator;
+import com.facebook.stetho.inspector.elements.StyleInfo;
 
 import javax.annotation.Nullable;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 final class ViewDescriptor extends AbstractChainedDescriptor<View> implements HighlightableDescriptor {
   private static final String ID_ATTRIBUTE_NAME = "id";
+  private static final Set<Method> viewGetters;
 
   private final MethodInvoker mMethodInvoker;
 
+  static {
+    Set<String> blacklistedGetters = new HashSet<>(
+      Arrays.asList(
+        "getAccessibilityCursorPosition",
+        "getAccessibilityWindowId",
+        "getAccessibilityViewId",
+        "getCameraDistance",
+        "getDrawingCacheBackgroundColor",
+        "getDrawingCacheQuality",
+        "getDrawingTime",
+        "getFilterTouchesWhenObscured",
+        "getFitsSystemWindows",
+        "getLabelFor",
+        "getLayerType",
+        "getNextFocusDownId",
+        "getNextFocusForwardId",
+        "getNextFocusLeftId",
+        "getNextFocusRightId",
+        "getNextFocusUpId",
+        "getOverScrollMode",
+        "getSystemUiVisibility"
+      ));
+
+    Method[] viewMethods = View.class.getDeclaredMethods();
+    List<Method> getters = new ArrayList<>();
```

Comment:
why not just have this be a `HashSet` ? otherwise we're populating an `ArrayList` and then copying it into a `HashSet` and wrapping it into an unmodifiable `Set`. if this is going to be a `Set` at the end, and since we don't need `List` semantics (right?), we may as well just skip the middle man